### ISSUE TITLE
Docker cloudbuilder folder copy issues

### ISSUE THREAD

**Issue by ekoome** [GoogleContainerTools/skaffold#442]
Issues without logs and details are more complicated to fix.
Please help us!

### Expected behaviour
Docker COPY command should copy folder and contents to docker image.

### Actual behaviour
Step 2/7 : COPY bin /app/bin
COPY failed: stat /var/lib/docker/tmp/docker-builder232188484/bin: no such file or directory

### Information
Confirmed that the folder is part of context generated by Skaffold (context.tar.gz) but unable to COPY the folder and contents using COPY command.

```
Skaffold version: 0.4.0
Operating system: Windows 10
Content of skaffold.yaml:
apiVersion: skaffold/v1alpha2
kind: Config
build:
  artifacts:
  - imageName: gcr.io/my-project/dtelapi
    workspace: .
deploy:
  kubectl:
    manifests:
      - kubernetes/test/deployment.yaml
profiles:
  - name: gcb
    build:
      googleCloudBuild:
        projectId: my-project

Docker File:

FROM node:8.6.0-alpine

COPY bin /app/bin
COPY build /app/build
COPY package.json npm-shrinkwrap.json /app

WORKDIR /app

RUN npm install

CMD ["npm", "start"]

```

### Steps to reproduce the behavior

  1. create basic project with folder
  2. copy folder and contents to Cloud builder docker context

**Comment by ekoome** [comment]
Figured out the problem is that the context is generated from windows and files are extracted as folder\folder\file in docker image. How do I force the context to unix format?

**Comment by masduo** [comment]
@dlorenc is this merge available in releases yet? the latest is [v0.5.0](https://github.com/GoogleContainerTools/skaffold/releases/tag/v0.5.0) which still fails on windows:
```
Step 3/22 : COPY ./src /app/src
Error: running skaffold steps: build: build step: running build for artifact: running build: COPY failed: stat /var/lib/docker/tmp/docker-builderxxx/src: no such file or directory
```

**Comment by dgageot** [comment]
@masduo Have you tried 0.9.0?

**Comment by masduo** [comment]
@dgageot just tried and i can confirm that the issue is fixed in the latest version 0.10.0, thanks
